docs/api/tray.md

```diff
@@ -327,6 +335,10 @@ Returns [`Rectangle`](structures/rectangle.md)
 
 The `bounds` of this tray icon as `Object`.
 
+#### `tray.getGUID()` _macOS_ _Windows_
```

Comment:
If my suggestion to use `identifier` is accepted, then I think we'd want this to become `getIdentifier` etc.